js/common.js

```diff
@@ -34,6 +33,7 @@ function loadOptions() {
     options.enableGalleries = options.hasOwnProperty('enableGalleries') ? options.enableGalleries : true;
     options.ambilightEnabled = options.hasOwnProperty('ambilightEnabled') ? options.ambilightEnabled : false;
     options.disabledPlugins = options.hasOwnProperty('disabledPlugins') ? options.disabledPlugins : [];
+    options.captionLocation = options.hasOwnProperty('captionLocation') ? options.captionLocation : 'below';
```

Comment:
This is not backwards compatible. All people who had disabled captions would suddenly be opted-in to these captions again. Please change `below` to either `below` or `none` depending on the existing `showCaptions` value.